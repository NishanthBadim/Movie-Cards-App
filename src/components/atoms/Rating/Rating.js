import React from 'react';

const Rating = (props) => {
    return (
        <div className="card-footer-badge float-right badge badge-primary badge-pill">
            {props.rating}
        </div>
    );
}
export default Rating;