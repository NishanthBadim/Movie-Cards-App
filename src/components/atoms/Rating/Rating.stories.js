import React from 'react'
import { withKnobs, number } from '@storybook/addon-knobs/react'
import Rating from './Rating'

export default {
    component: Rating,
    title: "Rating",
    decorators: [withKnobs]
}

export const Default = () => {
    return (
        <Rating rating={number("rating", 4.2)} />
    )
}