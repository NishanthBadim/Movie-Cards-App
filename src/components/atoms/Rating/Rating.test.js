import React from 'react';
import {create} from 'react-test-renderer'
import "@testing-library/jest-dom/extend-expect";
import Rating from './Rating'

test("Match the snapshot - Rating",() => {
    const input=create(<Rating />);
    expect(input.toJSON()).toMatchSnapshot();
})