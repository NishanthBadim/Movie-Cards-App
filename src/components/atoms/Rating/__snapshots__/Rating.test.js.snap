// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot - Rating 1`] = `
<div
  className="card-footer-badge float-right badge badge-primary badge-pill"
/>
`;
